### PR TITLE
Fix various problems (multiple commits)

### PR DESCRIPTION
**Pull request by GuySartorelli**
Fixes various problems I found while running the standardiser:

1. There was a syntax error using a heredoc instead of a nowdoc (`${` was interpreted as being for string interpolation, though it should just be included as-is)
2. New action was being added to a couple items that aren't supported (e.g. frameworktest), and _wasn't_ being added to docs
3. Processing was interrupted to tell me that frameworktest and a couple of other repos don't have a next-patch branch. That should just be a warning at the end, instead.

## Issue
- https://github.com/silverstripe/.github/issues/155

### CODE REVIEW

File: funcs_scripts.php [hunk 1]
```diff
@@ -263,7 +281,7 @@ function is_gha_repository()
  * Determine if the module being processed is "TOOLING"
  *
  * Example usage:
- * is_gha_repository()
+ * is_tooling()
```
Comment (GuySartorelli) [inline funcs_scripts.php:284]: This and the same below are unrelated fixes to phpdocs. Looks like it was just a copy/paste mistake

File: update_command.php [hunk 2]
```diff
@@ -219,5 +223,13 @@
     }
     output_repos_with_prs_created();
     output_prs_created();
+
+    // Report about any repos for which we couldn't find the right branch.
+    if (count($reposMissingBranch)) {
+        $reposString = implode("\n- ", $reposMissingBranch);
+        warning("Could not find branch to checkout for the following repos using --branch=$branchOption:\n- $reposString");
+        return Command::FAILURE;
+    }
```
Comment (GuySartorelli) [inline update_command.php:232]: Looks like this at the end of execution (after all other output):

image

**Review by emteknetnz — APPROVED** [review]
